server/src/pages/shot/view.js

```diff
@@ -296,9 +296,9 @@ class Body extends React.Component {
     if (typeof expireTime !== "number") {
       expireTime = expireTime.getTime();
     }
-    const deleteTime = new Date(expireTime + this.props.retentionTime);
+
     let restoreWidget;
-    const expirationTimeDiff = <TimeDiff date={deleteTime} />;
+    const expirationTimeDiff = <TimeDiff date={this.props.expireTime} />;
```

Comment:
This fixes below error in console for expired pages
error {"msg":"Warning: Failed prop type: Invalid prop `date` of type `date` supplied to `TimeDiff`, expected `number`.\n    in TimeDiff\n    in Body"}

Reply:
It seems like you're losing the `expireTime + retentionTime` logic here

Reply:
Didn't realise, we show restore button for 14 days on expired shots, will fix

Reply:
Need to account for restore logic which isn't toggle favorite but extend expiration date for retention time (14 days)